### ISSUE TITLE
Add build Fedora requirements to wiki

### ISSUE THREAD

**Issue by kallisti5** [scottdraves/electricsheep#46]
```sudo dnf install autoconf libtool gtk2-devel mesa-libGL-devel lua-devel libcurl-devel libxml-devel libjpeg-turbo-devel libgtop2-devel boost-devel boost-filesystem boost-thread freeglut-devel ffmpeg-devel tinyxml-devel flam3-devel```

**Comment by kallisti5** [comment]
Depends on #47

**Comment by SyedAmerGilani** [comment]
I needed also GLee-devel and wxGTK-devel

**Comment by Cybolic** [comment]
It seems wxGTK-devel >= 2.9 is now needed, which doesn't seem available in Fedora 27 :disappointed:

**Comment by Belsaros** [comment]
Followed Cybolic's instructions.
Still no sheep for me.

https://hastebin.com/labikoqera.sql
https://hastebin.com/vofivadezo.tex

~/.ssh/electricsheep/client_generic$ ./electricsheep-saver
./electricsheep-saver: line 1: exec: electricsheep: not found

**Comment by Cybolic** [comment]
The instructions I posted were just for building, not installing as well. Did you run `sudo make install` as well afterwards?

**Comment by Belsaros** [comment]
That wasit. That did the thing. Sorry.

**Comment by chadfurman** [comment]
update deps for Fedora 28?

**Comment by almereyda** [comment]
No, apparently not. @Cybolic's combined instructions above worked well, and will get simpler when #47 is merged.

```
sudo dnf install autoconf libtool gtk2-devel mesa-libGL-devel compat-lua-devel libcurl-devel libxml-devel libjpeg-turbo-devel libgtop2-devel boost-devel boost-filesystem boost-thread freeglut-devel ffmpeg-devel tinyxml-devel flam3-devel GLee-devel wxGTK3-devel lua-devel
git clone git@github.com:scottdraves/electricsheep.git && cd electricsheep/client_generic/
./autogen.sh
LUA_LIBS="-llua-5.1 -lm -ldl" ./configure
make
sudo make install
```

Given that flam3 is already part of the fedora distribution, has anyone ever considered providing a COPR build, or offering to maintain an official one?
https://copr.fedorainfracloud.org/coprs/fulltext/?fulltext=electricsheep

**Comment by bostrt** [comment]
I had to install `gcc-c++` in order to get around:
```
checking for libboost >= 1.39... configure: error: libboost version 1.41 or higher is required to compile electricsheep.
```

**Comment by Dionio** [comment]
Hi there,
I'm facing multiple compile errors like `../DisplayOutput/OpenGL/ShaderGL.cpp:41:3: error: ‘glDeleteObjectARB’ was not declared in this scope` during `make` on Fedora 29

I've installed all the packages mentioned here and in the main page https://github.com/scottdraves/electricsheep/wiki/Compiling and

Current suspect is some of GL *.h file with extensions is implicitly not included, but I'm not a C/C++ developer.

Did anyone faced such issue? Was you able to fix it?

**Comment by kallisti5** [comment]
Oh wow, my Lua fix was never merged.... It's been over two years  :-/

**Comment by sftoolhk** [comment]
Hi there,
I'm facing multiple compile errors like ../DisplayOutput/OpenGL/ShaderGL.cpp:41:3: error: ‘glDeleteObjectARB’ was not declared in this scope during make on Fedora 31
